Makefile

```diff
@@ -66,33 +68,34 @@ help:
-	@echo "- clean            Remove generated files"
-	@echo "- cleanall         Remove all the build artefacts"
-	@echo "- deploydev        Deploys current github master to dev. Specify SNAPSHOT=true to create snapshot as well."
-	@echo "- deployint        Deploys snapshot specified with SNAPSHOT=xxx to int."
-	@echo "- deployprod       Deploys snapshot specified with SNAPSHOT=xxx to prod."
-	@echo "- deploydemo       Deploys snapshot specified with SNAPSHOT=xxx to demo."
-	@echo "- deletebranch     List deployed branches or delete a deployed branch (BRANCH_TO_DELETE=...)"
-	@echo "- deploybranch     Deploys current branch to test (note: takes code from github)"
-	@echo "- deploybranchint  Deploys current branch to test and int (note: takes code from github)"
-	@echo "- deploybranchdemo Deploys current branch to test and demo (note: takes code from github)"
-	@echo "- ol               Update ol.js and ol-debug.js "
-	@echo "- translate        Generate the translation files (requires db user pwd in ~/.pgpass: dbServer:dbPort:*:dbUser:dbUserPwd)"
-	@echo "- help             Display this help"
+	@echo "- prod               Build app for prod (/prd)"
+	@echo "- dev                Build app for dev (/src)"
+	@echo "- lint               Run the linter"
+	@echo "- lintpy             Run the linter for the python files"
+	@echo "- autolintpy         Run the auto-corrector for python files"
+	@echo "- testdev            Run the JavaScript tests in dev mode"
+	@echo "- testprod           Run the JavaScript tests in prod mode"
+	@echo "- teste2e            Run browserstack and saucelabs tests"
+	@echo "- browserstack       Run browserstack tests"
+	@echo "- saucelabs          Run saucelabs tests"
+	@echo "- saucelabssingle    Run saucelabs tests but only with single platform/browser"
+	@echo "- apache             Configure Apache (restart required)"
+	@echo "- fixrights          Fix rights in common folder"
+	@echo "- all                All of the above (target to run prior to creating a PR)"
+	@echo "- clean              Remove generated files"
+	@echo "- cleanall           Remove all the build artefacts"
+	@echo "- deploydev          Deploys current github master to dev. Specify SNAPSHOT=true to create snapshot as well."
+	@echo "- deployint          Deploys snapshot specified with SNAPSHOT=xxx to int."
+	@echo "- deployprod         Deploys snapshot specified with SNAPSHOT=xxx to prod."
+	@echo "- deploydemo         Deploys snapshot specified with SNAPSHOT=xxx to demo."
+	@echo "- deletebranch       List deployed branches or delete a deployed branch (BRANCH_TO_DELETE=...)"
+	@echo "- deploybranch       Deploys current branch to test (note: takes code from github)"
+	@echo "- deploybranchint    Deploys current branch to test and int (note: takes code from github)"
+	@echo "- deploybranchdemo   Deploys current branch to test and demo (note: takes code from github)"
+	@echo "- ol                 Update ol.js and ol-debug.js"
```

Comment:
Not related, but this target is now called ol3cesium, and it updates ol3cesium, ol3 and cesium libraries.